packages/app/src/app/pages/Dashboard/Content/routes/Repositories/index.tsx

```diff
@@ -85,6 +92,30 @@ export const RepositoriesPage = () => {
         </Element>
       ) : null}
 
+      {!allowsPrivateRepos ? (
+        // && !isGHBannerDismissed
```

Comment:
We can maybe call this `permissionsBanner` instead of GH. Just a nitpick, not a required change if you disagree! Permissions is a longer word though.